custom/templates/DefaultRevamp/reactions_modal.tpl

```diff
@@ -1,4 +1,4 @@
-<div class="ui menu" style="overflow: scroll;">
```

Comment:
Pretty sure this was there to gracefully handle having a few dozen reactions nicely, might be misremembering. You can use the seeder to check though (edit the forum reaction count constant to get lots of reactions)